experimentation/dashboards/experiment_enrollments.dashboard.lookml

```diff
@@ -1479,6 +1283,227 @@
     col: 12
     width: 12
     height: 7
+  - title: Cumulative Crash Counts
+    name: Cumulative Crash Counts
+    model: experimentation
+    explore: experiment_crash_rates
+    type: looker_line
+    fields: [sum_of_crash_count, experiment_crash_rates.window_start_time]
+    filters:
+      experiment_crash_rates.experiment: new-users-sidebar-genai
+      experiment_crash_rates.window_start_date: 28 days
```

Comment:
Can we have it listen to the dashboard's filters instead of defaulting to these?